### PR TITLE
Feature 2232 eslint integration

### PR DESCRIPTION
**Pull request by mvolkmann**

### CODE REVIEW

File: web-ui/eslint.config.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import globals from "globals";
```
Comment (mvolkmann) [inline web-ui/eslint.config.js:1]: It took a few hours to arrive at this config!

File: web-ui/package.json [hunk 2]
```diff
@@ -51,7 +51,8 @@
     "build-storybook": "storybook build",
     "build": "vite build",
     "coverage": "vitest run --coverage --watch=false",
-    "format": "prettier --write 'src/**/*.{css,html,js,jsx}'",
+    "format": "prettier --write 'src/**/*.{css,html,js,jsx,ts,tsx}'",
+    "lint": "eslint --fix --quiet",
```
Comment (mvolkmann) [inline web-ui/package.json:55]: New script to run ESLint.

File: web-ui/src/components/reviews/periods/ReviewPeriods.jsx [hunk 3]
```diff
@@ -364,9 +364,9 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
         ) : periods.length > 0 ? (
           periods
             .sort((a, b) => {
-              return !!a.open === !!b.open
+              return Boolean(a.open) === Boolean(b.open)
```
Comment (mvolkmann) [inline web-ui/src/components/reviews/periods/ReviewPeriods.jsx:367]: ESLint likes this better.

File: web-ui/src/components/skeleton_loader/SkeletonLoader.test.jsx [hunk 4]
```diff
@@ -2,41 +2,35 @@ import React from 'react';
 import renderer from 'react-test-renderer';
 import SkeletonLoader from './SkeletonLoader';
 
-it("renders correctly when 'team' is passed as prop ", () => {
```
Comment (mvolkmann) [inline web-ui/src/components/skeleton_loader/SkeletonLoader.test.jsx:5]: ESLint removed the extra space at the ends of these description strings.

File: web-ui/src/components/skeleton_loader/SkeletonLoader.test.jsx [hunk 5]
```diff
@@ -2,41 +2,35 @@ import React from 'react';
...
-it("renders correctly when 'feedback_requests' is passed as prop ", () => {
+it("renders correctly when 'feedback_requests' is passed as prop", () => {
   const component = renderer.create(
     <SkeletonLoader type="feedback_requests" />
   );
   expect(component.toJSON()).toMatchSnapshot();
 });
 
-it("renders correctly when 'feedback_requests' is passed as prop ", () => {
+it("renders correctly when 'received_requests' is passed as prop", () => {
```
Comment (mvolkmann) [inline web-ui/src/components/skeleton_loader/SkeletonLoader.test.jsx:27]: The previous description string was repeated several times.

File: web-ui/src/components/skeleton_loader/SkeletonLoader.test.jsx [hunk 6]
```diff
@@ -2,41 +2,35 @@ import React from 'react';
...
   const component = renderer.create(
     <SkeletonLoader type="received_requests" />
   );
   expect(component.toJSON()).toMatchSnapshot();
 });
-it("renders correctly when 'feedback_requests' is passed as prop ", () => {
```
Comment (mvolkmann) [inline web-ui/src/components/skeleton_loader/SkeletonLoader.test.jsx:33]: This was a duplicate test.

File: web-ui/src/components/skeleton_loader/__snapshots__/SkeletonLoader.test.jsx.snap [hunk 7]
```diff
@@ -1,6 +1,6 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
-exports[`renders correctly when 'feedback_requests' is passed as prop  1`] = `
```
Comment (mvolkmann) [inline web-ui/src/components/skeleton_loader/__snapshots__/SkeletonLoader.test.jsx.snap:3]: This snapshot file was updated to match the test updates.

**Review by mkimberlin — APPROVED** [review]

**Review by jackkeller — APPROVED** [review]
Solid work! LGTM

**Pull request merged**